sdk/keyvault/tests.yml

```diff
@@ -6,6 +6,7 @@ extends:
     ServiceDirectory: keyvault
     MaxParallel: 5
     TimeoutInMinutes: 180
+    Clouds: 'Public,Preview,UsGov,China,Canary'
```

Comment:
I would also remove `Preview` from the `Clouds` list.

Reply:
Hi @benbp , I have removed `Preview` from the `Clouds` list.
Any other questions? If not, please merge it.